app/filters.tsx

```diff
@@ -108,14 +108,14 @@ export type CreatorFilter = {
 };
 
 export const creatorFilters: CreatorFilter[] = [
-  {
-    name: 'AI markets',
-    key: 'ai',
-    when: { creator_in: AI_AGENTS_ALLOWLIST },
-  },
   {
     name: 'All markets',
     key: 'all',
     when: {},
   },
+  {
+    name: 'AI markets',
```

Comment:
We don't need AI anymore here as it will have the svg

```suggestion
    name: 'markets',
```

Reply:
Tested it and didn't work as well as expected. Will add the AI logo as it is, and we can check this again in the future.